javaslang/src/main/java/javaslang/concurrent/Future.java

```diff
@@ -247,6 +249,38 @@
     }
 
     /**
+     * Creates a {@code Future} with the given java.util.concurrent.CompletableFuture, backed by the {@link #DEFAULT_EXECUTOR_SERVICE}
```

Comment:
We could also write `java.util.concurrent.CompletableFuture` as `{@code }` or `{@link }`...